doc/specifications/cache/multi_tagging.md

```diff
@@ -0,0 +1,126 @@
+# Multi Tagging
+
+
+_WIP @todo_
+
+
+
+
+## Description
+Being built on Symfony, one of the cornerstones of eZ Platform is that it has always extended Symfony HttpCache
+to better integrate with the CMS. Starting in version v2014.11 *(eZ Publish Platform 5.4)* this was further improved
+by building on top of FOSHttpCache.
+
+But even if we moved to FOSHttpCache, we still had our own Symfony Proxy Store, and still abstract cache clearing
+internally. Starting with kernel v6.5 *(eZ Platform v1.5)*, this allowed us to give HttpCache layer several changes to:
+1. Be able to support multiple tagging, across Varnish and Symfony Proxy, as opposed to just single location tagging.
+2. Allow usage of Varnish xkey VMOD for better performance with Varnish.
+
+In both cases this is needed to more reliably be able to clear cache, and to improve performance. At later point these
+features might appear in FOSHttpCache v2.0 in a more generic form for us to build upon.
+
+
+## Tags
+
+As the systems is extensible, important part of this is to document the used tags to avoid wrong use, or conflicts.
+If you add own tag types, please prefix with 1-3 letters abbreviating your *(company/full)* name.
+
+The tags are needs to be made in a way so that cache can be cleared using nothing but what info is available on signal,
+and the signals will need to be expanded to contain the relevant info depending on operation they correspond to.
+
+
+### Predefined System Tags
+
+
+- `content-<content-id>`
+
+*Tagging*: Used for tagging content responses with id.
+*Clearing*: When a operation is specifically affecting just given content.
+
+
+- `content-type-<content-type-id>`
```

Comment:
What about `content-type-group-<content-type-group-id>` ? It would be useful for list of content types by group, to expire those when a new content-type is added to the group.